_includes/author-profile.html

```diff
@@ -23,7 +23,7 @@ <h3 class="author__name">{{ author.name }}</h3>
     <button class="btn btn--inverse">Follow</button>
     <ul class="author__urls social-icons">
       {% if author.location %}
-        <li><i class="fa fa-fw fa-map-marker" aria-hidden="true"></i> {{ author.location }}</li>
+        <li class="author__location"><i class="fa fa-fw fa-map-marker" aria-hidden="true"></i> {{ author.location }}</li>
```

Comment:
Took a closer look and adding the `author__location` class to the `<li>` element is unnecessary. As is the `.author__location` declaration in `_sidebar.scss`.

If `cursor: default` is added to `.author__urls` [here](https://github.com/mmistakes/minimal-mistakes/blob/master/assets/_scss/_sidebar.scss#L155) you can get away with adding a single line and not having to mess with the sidebar's markup.